testsuite/MDAnalysisTests/analysis/test_diffusionmap.py

```diff
@@ -80,3 +80,15 @@ def test_transform(u, dmap):
     dmap.run()
     diffusion_space = dmap.transform(n_eigenvectors, 1)
     assert diffusion_space.shape == (eigvects.shape[0], n_eigenvectors)
+
+def test_long_traj(u):
+    with pytest.warns(UserWarning, match='The distance matrix is very large'):
+        dmap = diffusionmap.DiffusionMap(u)
+        dmap._dist_matrix.run()
+        dmap._dist_matrix.n_frames = 5001
+        assert dmap.run()
+
+def test_universe(u):
+    trj_only = u.trajectory
+    with pytest.raises(ValueError, match='U is not a Universe'):       
+        assert diffusionmap.DiffusionMap(trj_only)
```

Comment:
As above, you shouldn't need the `assert` here.

Reply:
Thanks @IAlibay, I was getting confused with the usage of assertions here but your comments helped me to get the point. Your feedback is very appreciated.